journal-temp/modernized/14-chapter-14.adoc

```diff
@@ -695,7 +774,7 @@ had great reason to be the more easy and thankful,
 that we had some relief from those extreme persecutions
 and hardships we had long suffered under.
 
-6+++.+++ Howbeit,
+5+++.+++ Howbeit,
```

Comment:
Good call switching this from 6 to 5, but you forgot to marked it a [.numbered] section 🙂